packages/sommelier.rb

```diff
@@ -109,6 +96,10 @@ def self.build
         # https://source.chromium.org/chromium/chromium/src/+/master:third_party/chromite/third_party/lddtree.py;drc=46da9a8dfce28c96765dc7d061f0c6d7a52e7352;l=146
         system "cat <<'EOF'> sommelier_sh
 #!/bin/bash
+function readlink(){
+  coreutils --coreutils-prog=readlink "$@"
```

Comment:
Doesn't doing this still require coreutils?

Reply:
`coreutils` is a built-in command for GNU based Linux (`/usr/bin/coreutils`)

Reply:
I have to agree with @supechicken on this.  We shouldn't need coreutils.

Reply:
If we're going to update, we might as well get the latest code?  https://chromium.googlesource.com/chromiumos/platform2.git/+/be7553c1d04f95cae6d6e7c3c2292fb0440ed2db

Reply:
> If we're going to update, we might as well get the latest code? 

Done

Reply:
Newer platform2 sommelier doesn't work due to issues with the virtual device not showing up when sommelier isn't tunneling through crosvm.

I have no problems with updating to newer git so long as you build and confirm sommelier works!

(Otherwise revert to the older version.). 😅

Reply:
Just check it in my system, it built successfully on chroot but failed on my system

Reply:
Should we revert or try an older git hash?

Reply:
The current version is the latest before dropping virtwl support and I can get it working